.config/htconfig.php

```diff
@@ -6,11 +6,15 @@
 // Use environment variables for mysql if they are set beforehand
 if (!empty(getenv('MYSQL_HOST'))
 	&& !empty(getenv('MYSQL_PORT'))
-	&& !empty(getenv('MYSQL_USERNAME'))
+	&& (!empty(getenv('MYSQL_USERNAME')) || !empty(getenv('MYSQL_USER')))
```

Comment:
Why are both versions cohabiting? Can't we just use one of them?

Reply:
Mariadb => MYSQL_USER
MySQL => MYSQL_USERNAME

I want to be open for mysql and mariadb installations :-)

Reply:
Shaking my head.

Reply:
mine too, took me a while to find it ;-)